breeze

```diff
@@ -629,7 +629,7 @@ function parse_arguments() {
         -k|--stop-environment)
           COMMAND_TO_RUN="run_docker_compose"
           DOCKER_COMPOSE_COMMAND="down"
-          export EXTRA_DC_OPTIONS+=("--remove-orphans")
```

Comment:
shouldn't it work for Mac too ?

Reply:
Not by default. MacOS comes with bundled 3.* series of bash (the current version is 5.*) due to bash moving from GPL2 to GPL3. Now they switched to zsh in catalina as default shell but default bash remains 3.* . it has a number of missing new syntax from modern bash :(

Reply:
in bash 3 export can only work with assignment not with +=.. It considers + as part of variable name in this case....